README.md

```diff
@@ -124,7 +124,18 @@ format manually.
 
 The utility program `nibconv` from
 [NIBTools](https://c64preservation.com/dp.php?pg=nibtools) can be used to
-convert between `.d64` and `.g64`. For Linux it is obtained and built as
+convert between `.d64` and `.g64`. 
+
+Usage example:
+
+```sh
+nibconv somedisk.d64 somedisk.g64
+```
+#### Obtaining nibtools
+
+##### Linux
+
+For Linux it is obtained and built as
 follows (for windows pre-built executables are available).
 
```

Comment:
Maybe the windows part could be under its own header now.

Reply:
And link to the pre-built binaries.

Reply:
Added